OneBusAway/otpkit
# Issue 22 - Origin Destinatoin Map Marker

# Origin Destination Map Marker, Limited to generate marker in SwiftUI Map

## Related Issues
#22 

## Description

This PR is limited to add generated marker on SwiftUI map. The functionality will be added later

## Media

https://github.com/user-attachments/assets/f465e090-3602-4fc7-8703-099d5ba47574

## Test Instructions

1. Enable users location
2. Try to tap the map, the marker should be visible and added

## Additional Information
The Origin Destination UI is moved to the bottom because it will block the Map Controls
